travis/update-release.py

```diff
@@ -16,6 +16,15 @@ def pull_image_and_get_sha(image_name_and_tag):
     return get_image_sha(image_name_and_tag)
 
 def get_image_sha(image_name_and_tag):
+    try:
+        result = subprocess.check_output(['docker', 'image', 'inspect', '--format', '{{index (split .Id ":") 1}}', image_name_and_tag], universal_newlines=True)
+    except subprocess.CalledProcessError as e:
+        # Handle any errors that occur during the subprocess execution
+        print("Error:", e)
+        return "error"
+    return result.strip()
+
+def get_repo_digest_sha(image_name_and_tag):
```

Comment:
good catch, so when should we use id-sha vs repo-digest sha, shouldn't we always use repo-digest sha?